CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,19 @@
 
 import Foundation
 
-print("Hello, World!")
+let myLottoNumbers: Set<Int> =  [1, 2, 3, 4, 5, 6]
+let numbers = Array(1...45)
+var lotto: Set<Int> = []
 
+    while lotto.count < 6 {
+        lotto.insert(numbers.randomElement()!)
```

Comment:
`randomElement()` 메서드는 왜 옵셔널을 반환할까요?
느낌표를 사용하지 않으려면 어떻게 코드를 작성할 수 있을까요?

Reply:
```suggestion
        lotto.insert(Int.random(in: 1...45))
```
구글링을 통해 느낌표를 사용하지 않고 코드를 작성해봤습니다!